packages/nodes/list/src/transforms/unwrapList.ts

```diff
@@ -16,6 +18,29 @@ export const unwrapList = <V extends Value>(
   editor: PlateEditor<V>,
   { at }: { at?: Path } = {}
 ) => {
+  const ancestorListTypeCheck = () => {
+    if (getAboveNode(editor, { match: { type: getListTypes(editor), at } })) {
+      return true;
+    }
+
+    // The selection's common node might be a list type
+    if (!at && editor.selection) {
+      const commonNode = getCommonNode(
```

Comment:
you only use commonNode[0]
why not just define that as the const with a more descriptive name that tells me what (type) commonNode[0] is